lib/resources.js

```diff
@@ -554,6 +564,46 @@ const buildElasticBody = function (params) {
   return body
 }
 
+/**
+ * Given params which include an identifier number, returns query for that identifier
+ */
+
+const queryForIdentifierNumber = function (params) {
+  if (params.lccn) {
+    return {
+      query: {
+        regexp: {
+          idLccn: {
+            value: `[^\\d]*${regexEscape(params.lccn)}[^\\d]*`
+          }
+        }
+      }
+    }
+  } else if (params.issn) {
+    return { query: { bool: { must: { term: { idIssn: params.issn } } } } }
```

Comment:
What's the reason for the `bool` structure here? You generally don't need `bool` unless you're doing boolean and/or queries or using `filter`. I think it can be simplified to just
```
    return { query: { term: { idIssn: params.issn } } }
```

Reply:
Thanks! I didn't know `bool` could be omitted. Fixed.